utils/inspect-response.js

```diff
@@ -1,6 +1,32 @@
 const explorer = require('./explorer');
-
+const config = require('../get-config')();
+const chalk = require('chalk');  // colorize output
 const util = require('util');
+
+
+const checkExistAccount = (error, options) => {
+    if(!String(error).includes('does not exist while viewing')) return false;
+
+    console.log(chalk`\n{bold.red Account {bold.white ${options.accountId}} is not found in {bold.white ${config.helperAccount}} network.\n}`);
+    
+    const re = new RegExp("[^\.]*$", "gi")
```

Comment:
@rozum-dev There is still several issues in this line ;)

`12:27  error  Strings must use singlequote quotes`
`12:30  error  Unnecessary escape character: \. no-useless-escape`
`12:38  error  Strings must use singlequote quotes`
`12:43  error  Missing semicolon semi`

Consider using `yarn run lint` to check for such issues and `yarn run fix` to fix them (most of them).